docs/features/offloading.md

```diff
@@ -1,10 +1,20 @@
 # Offloading
 
-**Workload offloading** is enabled by the **virtual node** abstraction.
-A virtual node is spawned at the end of the peering process in the local (i.e., consumer) cluster, and represents (and aggregates) the subset of resources shared by the remote cluster.
+**Workload offloading** is enabled by a **virtual node**, which is spawned in the local (i.e., consumer) cluster at the end of the peering process, and represents (and aggregates) the subset of resources shared by the remote cluster.
+
 This solution enables the **transparent extension** of the local cluster, with the new node (and its capabilities) seamlessly taken into account by the vanilla Kubernetes scheduler when selecting the best place for the workloads execution.
 At the same time, this approach is fully compliant with the **standard Kubernetes APIs**, hence allowing to interact with and inspect offloaded pods just as if they were executed locally.
 
+## Assigned resources
+
+By default, the virtual node is created by allocating 90% of the resources available (TODO: totally available or currently free?) on the remote cluster. For instance, if the remote cluster has 100 CPU cores, the virtual node will advertise 90 CPU cores.
+This applies to all the resources available in the remote cluster, e.g., CPUs, RAM, GPUs.
+TODO: is it possible to customize this value, e.g., to 50%? Is it possible to change it at run-time (e.g., from 90% to 80%)?
```

Comment:
Yes using the `--sharing-percentage` is possible to set the amount of shared resources. 
The reference Is [here](https://docs.liqo.io/en/v0.7.2/installation/install.html#control-plane). It is possible to change the value at runtime.